### DISCUSSION TITLE
WWDC24 Week Session Claims

### DISCUSSION THREAD

**Discussion by Jeehut** [WWDCNotes/WWDCNotes#1] [Announcements]
This discussion is just here to prevent duplicate work or avoid PR conflicts during the week of WWDC24 or the week after. If you read this at a later time, then please ignore this and just go ahead with your plans to provide notes.

But if WWDC24 just recently happened, it's possible someone else wants to provide notes for the same session as you at the same time. So please add a comment in this discussion with the full title(s) of the session(s) you want to cover. And search in this thread if you want to know if someone else is working on a session you're also interested in.

If the comment is more than 2 days old and there's still no note online and also no PR waiting to be merged, then please leave a reply that you're taking over the notes for the session and go ahead.

Please note that there's no rule that a session can only be covered by one person. Quite the opposite: Inputs from multiple people can only improve the notes. So feel free to reply on a comment to get in touch with the other person.

**Comment by pitt500** [comment]
Hi @Jeehut, I would like to work on "What’s new in Swift" https://wwdcnotes.github.io/WWDCNotes/documentation/wwdcnotes/wwdc24-10136-whats-new-in-swift/

If nobody has taken it yet, I would like to help with this one :)

**Reply by MortenGregersen** [reply]
I have already made notes for this: https://github.com/WWDCNotes/WWDCNotes/pull/4

**Reply by pitt500** [reply]
@Jeehut What should we do here? 😅

**Reply by Jeehut** [reply]
@pitt500 Try to merge with Morten's file, I think, if you can. We can't prevent people taking notes for themselves for sessions they are going to watch anyways and then providing them after. Especially for the "basic" sessions many people are going to watch… this discussion probably gonna be most helpful for like the 115 sessions that are not the basic ones, but the "What's New" sessions for Swift, SwiftUI, Xcode, etc. are quite popular (for a reason).

**Reply by pitt500** [reply]
Makes totally sense. Thank you

**Reply by MortenGregersen** [reply]
I wasn't aware of this discussion before I took the notes, and started with this, as I think it is a basis for the rest of the SwiftUI/SwiftData sessions.

**Comment by halmueller** [comment]
I'm starting 10166 Build compelling spatial photo...experiences

**Reply by halmueller** [reply]
Merged.

**Comment by MortenGregersen** [comment]
I am starting "10144 - What's new in SwiftUI"

**Reply by MortenGregersen** [reply]
Done: #5

**Comment by MortenGregersen** [comment]
I am starting "10135 - What's new in Xcode"

**Reply by MortenGregersen** [reply]
Done: #6

**Comment by MortenGregersen** [comment]
I am starting "10137 - What's new in SwiftData"

**Reply by MortenGregersen** [reply]
Done: #7

**Comment by RamitSharma991** [comment]
Hello ! 
I have 10183 (What's new in create ML) ready , can i claim it. 
Thanks.

**Reply by RamitSharma991** [reply]
Successfully Merged!

**Comment by NinjaLikesCheez** [comment]
Finished 10210: Bring your app's core features to users with App Intents (https://github.com/WWDCNotes/WWDCNotes/pull/14)

(yes, I forgot to claim it, my bad my bad)

**Comment by coughski** [comment]
I'll claim 10151: `Create custom visual effects with SwiftUI`

**Reply by coughski** [reply]
This is pretty much done, but I don't have good enough WiFi to upload it yet

**Comment by RamitSharma991** [comment]
Claiming 10161 - Deploy machine learning .... with core ML
Thanks!

**Comment by petermolnar-dev** [comment]
Would love to start with "Meet Swift Testing".

**Reply by petermolnar-dev** [reply]
PR is opened here: [21](https://github.com/WWDCNotes/WWDCNotes/pull/21)

**Comment by teacup** [comment]
Claiming: 10157 - Extend your app’s controls across the system

**Reply by teacup** [reply]
Done: https://github.com/WWDCNotes/WWDCNotes/pull/16

**Comment by feifanzhou** [comment]
I'll work on `10149`: "Work with windows in SwiftUI"

**Reply by feifanzhou** [reply]
https://github.com/WWDCNotes/WWDCNotes/pull/23

**Comment by petermolnar-dev** [comment]
Claiming: WWDC24-10195-Go-further-with-Swift-Testing

**Reply by dustynaugust** [reply]
@petermolnar-dev @Jeehut 
Are you two still working on this? I have some personal notes that I could enhance for WWDCNotes

**Reply by Jeehut** [reply]
@dustynaugust The comment has been 2 days ago. At this point, feel free to provide your notes first, @petermolnar-dev can then update your notes.

**Reply by petermolnar-dev** [reply]
Agreed with @Jeehut , I am at 33%, but needs to catch up with work. Will have time tomorrow (probably for some updates), but all yours @dustynaugust ;).

**Reply by dustynaugust** [reply]
Thanks @Jeehut. 

@petermolnar-dev Let me know if you don't make any progress today! There are plenty more sessions that need notes. 💪🏻

**Reply by petermolnar-dev** [reply]
Thanks @dustynaugust, I am around 75%, so definitely made progress. Will try to finish tomorrow. There are plenty small hints along the way, so that’s why I am progressing slowly.

**Reply by petermolnar-dev** [reply]
Wasn't easy, but here it goes: https://github.com/WWDCNotes/WWDCNotes/pull/34 . @dustynaugust , If you see we need to add something, feel free to jump in.

**Comment by Jeehut** [comment]
I'm finally heading back home from California. I plan to watch & summarize the following 2 sessions during my flight:

* What’s new in SF Symbols 6
* Demystify SwiftUI containers

**Reply by Jeehut** [reply]
Merged in https://github.com/WWDCNotes/WWDCNotes/pull/32. 🎉 

Note that I was too sleepy to do 10 sessions, so I did only the first two. I edited the comment accordingly.

**Comment by RamitSharma991** [comment]
Claiming  WWDC24 - 10125 - Streamline sign-in with passkey upgrades and credential managers.
Thanks!

**Reply by RamitSharma991** [reply]
PR ✅

**Comment by dustynaugust** [comment]
Starting on 10150 SwiftUI essentials

**Reply by dustynaugust** [reply]
PR opened: #28

**Comment by dustynaugust** [comment]
Claiming **10181 Xcode essentials**

**Reply by dustynaugust** [reply]
PR opened: #28

**Comment by halmueller** [comment]
PR pending: 10096-Design-interactive-experiences-for-visionOS

**Comment by dustynaugust** [comment]
Claiming 10184 A Swift Tour Explore: Swifts features and design

**Comment by Jeehut** [comment]
Starting to summarize "Migrate your app to Swift 6"…

**Reply by Jeehut** [reply]
Done: 69118941a66d2d8b1ae63adb1190105da8a41ffe

**Comment by RamitSharma991** [comment]
Claim: 10204 "Build a great Lock screen camera capture experiene"

**Reply by RamitSharma991** [reply]
PR ✅

**Comment by nikolaikhud** [comment]
Made a PR on 10145 "Enhance your UI animations and transitions" today.

**Comment by VielfaltSoftware** [comment]
Made a PR for 10103 "Discover RealityKit APIs for iOS, macOS and visionOS"
